modules/oe_whitelabel_contact_forms/oe_whitelabel_contact_forms.module

```diff
@@ -74,13 +74,11 @@ function oe_whitelabel_contact_forms_preprocess_status_messages(&$variables) {
       if ($field->isEmpty() || !$field->access()) {
         continue;
       }
-      $value = 'value';
-      if ($field->getFieldDefinition()->getType() == 'skos_concept_entity_reference') {
-        $value = 'target_id';
-      }
       $items[] = [
         'term' => $field->getFieldDefinition()->getLabel(),
-        'definition' => $field->first()->getValue()[$value],
+        'definition' => $field->getFieldDefinition()->getType() === 'skos_concept_entity_reference'
+        ? $field->referencedEntities()
+        : $field->first()->getValue()['value'],
```

Comment:
Wrong indentation. I know it is PhpStorm's fault but we should fix manually.
```suggestion
        'definition' => $field->getFieldDefinition()->getType() === 'skos_concept_entity_reference'
          ? $field->referencedEntities()
          : $field->first()->getValue()['value'],
```

Reply:
Also, by passing a Concept entity to the template, the template will implicitly call `->label()` on that entity, through this line in bcl-descritpion-list.html.twig:
```twig
                {{- _definition.label -}}
```
This is how it works: If the value is an object, then it will try `$obj->$key` and then `$obj->$key()`.
See `twig_get_attribute()`. Crazy, eh?

Let's not rely on this magic.
Instead, let's do something like this:
(please don't copy this 1:1, it is only for the rough direction)

```php
/** @var \Drupal\rdf_skos\Entity\Concept[] $concept_entities */
$concept_entities = $field->referencedEntities();
if (!isset($concept_entities[0])) {
  continue;
}
$items[] = [
  'term' => $field->getFieldDefinition()->getLabel(),
  'definition' => [['label' => $concept_entities[0]->label()]],
];
```

Reply:
I think it is complex enough that we should use if/else instead of a ternary.